src/avfoundation.cs

```diff
@@ -796,31 +797,41 @@ interface AVAudioFormat : NSSecureCoding {
 
 		[Export ("isEqual:"), Internal]
 		bool IsEqual (NSObject obj);
+		
+		[iOS (10,0), TV (10,0), Watch (3,0), Mac (10,12)]
+		[NullAllowed, Export ("magicCookie", ArgumentSemantic.Retain)]
+		NSData MagicCookie { get; set; }
 	}
 
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	interface AVAudio3DMixing {
+		[NoWatch]
```

Comment:
I need to test that removing the entire class is not an issue, that is the mean reason. The headers do it for each method and not the class, while in others they do use it in the class. My concern, there are methods available in the watch that do take the class AND are available in the watch,